sjb/config/test_cases/test_branch_openshift_ansible_logging.yml

```diff
@@ -121,8 +121,15 @@ extensions:
                          --become-user root         \
                          --connection local         \
                          --inventory sjb/inventory/ \
-                         -e containerized=true      \
-                         -e deployment_type=origin  \
+                         -e openshift_logging_install_logging=False \
+                         -e openshift_deployment_type=origin  \
+                         -e debug_level=2           \
```

Comment:
Is this `--loglevel` for Origin components? Let's keep that at 5

Reply:
This matches the 'install origin' step; that value is unchanged in this PR.

Reply:
Hrm.

Reply:
Can you make a follow-up to bump it to 5? Debugging at 2 is hard if not impossible

/cc @richm